examples/volano/kit/volano.py

```diff
@@ -1,26 +1,22 @@
 #!/usr/bin/env python3
 # Copyright (C) 2024 Vrije Universiteit Brussel. All rights reserved.
 # SPDX-License-Identifier: MIT
-
 """
 Benchkit support for Volano
 """
 
-import os
 import pathlib
 import re
-import subprocess
-import time
 from typing import Any, Dict, Iterable, List, Optional
 
 from benchkit.benchmark import Benchmark, CommandAttachment, PostRunHook, PreRunHook
 from benchkit.campaign import CampaignCartesianProduct, Constants
-from benchkit.commandattachments.tcpdump import TcpDump
 from benchkit.commandwrappers import CommandWrapper
 from benchkit.dependencies.packages import PackageDependency
 from benchkit.platforms import Platform
 from benchkit.sharedlibs import SharedLib
 from benchkit.utils.types import PathType
+from benchkit.utils.dir import caller_dir
 
 
 class CounterBenchmark(Benchmark):
```

Comment:
Could you change the class name to something more meaningful?

Reply:
suggestion?

Reply:
VolanoBenchmark, This was an error on my side way back.

Reply:
done